app/scripts/modules/core/pipeline/config/triggers/travis/travisTriggerOptions.component.ts

```diff
@@ -65,17 +65,22 @@ export class TravisTriggerOptions {
   };
 }
 
+class TravisTriggerOptionsComponent implements IComponentOptions {
+  public bindings: any;
+  public templateUrl: string;
+  public controller: any;
+
+  constructor() {
+    this.bindings = {
+      command: '='
+    };
+    this.templateUrl = require('./travisTriggerOptions.component.html');
+    this.controller = TravisTriggerOptions;
+  }
+}
```

Comment:
You should use just that string for `selectorTemplate` in `travisTrigger.module.ts`. Using here makes the component call itself recursively. I'll think you'll have to `require` your template in the `IComponentOptions` or inline the whole thing.